spec/models/user_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+RSpec.describe User, type: :model do
+  context 'validations' do
+    it { should validate_presence_of(:name) }
+    it { should validate_presence_of(:address) }
+    it { should validate_presence_of(:password) }
+    it { should validate_presence_of(:phone) }
```

Comment:
has_many book

Reply:
fix when assign the book to user, later task